packages/t/tinyxml2/xmake.lua

```diff
@@ -0,0 +1,40 @@
+package("tinyxml2")
+
+    set_homepage("http://www.grinninglizard.com/tinyxml2/")
+    set_description("simple, small, efficient, C++ XML parser that can be easily integrating into other programs.")
+
+    add_urls("https://github.com/leethomason/tinyxml2/archive/8.0.0.tar.gz", {alias = "tar"})
+    add_urls("https://github.com/leethomason/tinyxml2.git", {alias = "git"})
+    add_versions("git:8.0.0", "bf15233ad88390461f6ab0dbcf046cce643c5fcb")
+    add_versions("tar:8.0.0", "6ce574fbb46751842d23089485ae73d3db12c1b6639cda7721bf3a7ee862012c")
```

Comment:
xmake will checkout  8.0.0 tag for git url.

and it will use tar url first.

```lua  add_urls("https://github.com/leethomason/tinyxml2/archive/8.0.0.tar.gz")
add_urls("https://github.com/leethomason/tinyxml2.git")
    add_versions("8.0.0", "6ce574fbb46751842d23089485ae73d3db12c1b6639cda7721bf3a7ee862012c")
```